test/bootsample-non-parametric.jl

```diff
@@ -14,13 +14,13 @@ function test_bootsample(bs, ref, raw_data, n)
     t0 = original(bs)
     @fact length(t0) --> length(ref)
     [@fact t --> roughly(r) for (t, r) in zip(t0, ref)]
-    
+
```

Comment:
I would remove the whitespace changes in the PR (I'll convert the entire test suite to `Base.Test` once we have merged this).